src/components/store/pages/checkout/index.js

```diff
@@ -8,10 +8,10 @@ import ShippingStep from "./steps/shipping";
 import PaymentStep from "./steps/payment";
 import useCart from "../../../../hooks/cart";
 
-function cartItem(item, cart) {
+function cartItem(item) {
   return (
-    <>
-      <tr className="product" key={item.slug}>
+    <Fragment key={item.slug}>
```

Comment:
with latest React 16 you don't need to import `Fragment`, you can just do `<>...</>`

Reply:
I tried to use just use the shorthand fragment, but it would not allow me to add an attribute for the collection items key.